docs/kpod-logout.1.md

```diff
@@ -0,0 +1,49 @@
+% kpod(1) kpod-logout - Simple tool to logout of a registry server
+% Urvashi Mohnani
+# kpod-logout "1" "August 2017" "kpod"
+
+## NAME
+kpod-logout - Logout of a container registry
+
+## SYNOPSIS
+**kpod logout**
+**REGISTRY**
+[**--help**|**-h**]
+
+## DESCRIPTION
+**kpod logout** logs out of a specified registry server by deleting the credentials
+stored in the **auth.json** file. The path of the authentication file can be overrriden by the user by setting the **infile** flag. The default path used is **XDG\_RUNTIME\_DIR/containers/auth.json**.
+
+**kpod [GLOBAL OPTIONS]**
+
+**kpod logout [GLOBAL OPTIONS]**
+
+**kpod logout [OPTIONS] SERVER [GLOBAL OPTIONS]**
+
+## OPTIONS
+
+**--infile, -i**
+Path of the authentication file. Default is XDG_\RUNTIME\_DIR/containers/auth.json
+
+## GLOBAL OPTIONS
```

Comment:
Remove --help is not a global option.

Reply:
Have I been doing man pages wrong all this while? I thought --help was a global option?